README.md

```diff
@@ -61,6 +61,20 @@ To activate Poetry's virtual environment, run:
 
     poetry shell
 
+### Setting up Wagtail guide to develop on Docker
```

Comment:
```suggestion
### Setting up development with Docker
```

Nitpicky - just removing the name of this project in the title - not really needed.